lib/util.js

```diff
@@ -2,8 +2,12 @@
 
 const _ = require('lodash')
 const call = require('call')
+const Controller = require('trails-controller')
+const Policy = require('trails-policy')
 const lib = require('.')
 
+const isClassStandardized = {} // cache to know if class is standar or not
```

Comment:
This all looks good except I don't like storing state in a module-global variable. Also, indexing by `className` isn't reliable. The whole `api.x.y.z` namespace needs to be used.